src/items/controllers/history.controller.ts

```diff
@@ -0,0 +1,22 @@
+import { Body, Controller, Get, Param, Post } from '@nestjs/common';
+import { Public } from '../../auth/decorators/public.decorator';
+import { HistoryRequestDto } from '../dto/history-request.dto';
+import { History } from '../entities/history.entity';
+import { HistoryService } from '../services/history.service';
+
+@Controller('history')
+export class HistoryController {
+  constructor(private historyService: HistoryService) {}
+
+  @Public()
+  @Get('items/:itemId')
```

Comment:
I will argue on this one. I think that according to the rest nomenclature it makes more sense having something like 'api/items/123/history' than having 'api/history/items/123'. What do you think?

Reply:
It's better